src/domains/community/write/page/WriteCommunityPostPage.jsx

```
@@ -3,8 +3,15 @@ import LastFormLine from '../../../job/common/components/LastFormLine'
 import Button from '../../../../components/web/Button'
 import WriteCommunityPostForm from '../components/WriteCommunityPostForm'
 import PinkButton from '../../../../components/web/PinkButton'
+import { useNavigate } from 'react-router-dom'
+import ROUTER_PATHS from '../../../../routes/RouterPath'
 
 const WriteCommunityPost = () => {
+  const navigate = useNavigate()
+
+  const handleSaveDraft = (draftId) => {
```

ESLint

[error] 'draftId' is defined but never used.

(no-unused-vars)